Shared/ThresholdKeyView.swift

```diff
@@ -216,15 +235,21 @@ struct ThresholdKeyView: View {
                         Spacer()
                         Button(action: {
                             Task {
-                                let question = "what's your password?"
-                                let answer = "blublublu"
-                                _ = try! await SecurityQuestionModule.change_question_and_answer(threshold_key: threshold_key, questions: question, answer: answer)
-                                let key_details = try! threshold_key.get_key_details()
-                                totalShares = Int(key_details.total_shares)
-                                threshold = Int(key_details.threshold)
+                                    let question = "what's your password?"
+                                    let answer = "blublublu"
+                                    let result = try! await SecurityQuestionModule.change_question_and_answer(threshold_key: threshold_key, questions: question, answer: answer)
```

Comment:
This can fail and others not part of this PR

Reply:
my bad. Checked all buttons again and catched errors